torch/testing/_internal/common_distributed.py

```diff
@@ -144,7 +144,7 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 TIMEOUT_DEFAULT = 100
-TIMEOUT_OVERRIDE = {}
+TIMEOUT_OVERRIDE = {"test_ddp_uneven_inputs": 600}
```

Comment:
That seems like a pretty long timeout, how long does this test take? Ideally, our tests should be much faster than this.

Reply:
It was timing out in the CI at a 300s timeout, but it work fined under stresstests in a local OSS build, and now the CI seems to pass after I bumped the timeout. Locally, the slowest run of this test only took ~110s.